src/fun/anon_channel/anon.go.tmpl

```diff
@@ -0,0 +1,31 @@
+{{/*
+	Sticky message with button to submit an anonymous message.
+	See <https://yagpdb-cc.github.io/fun/anon-channel/main-cc> for more information.
+
+	Author: SoggySaussages <https://github.com/SoggySaussages>
+*/}}
+
+{{ $all_anon := true }}
```

Comment:
Personally I like having configuration values in uppercase, but given that we don't have a formal requirement for this, this is just a stylistic remark from me.